src/Aquarius.ONE.ClientSDK/Enterprise/Authentication/AuthenticationApi.cs

```diff
@@ -388,12 +388,12 @@ public bool IsAuthenticated
             }
         }
 
-        public void SetToken(string accessToken, DateTimeOffset expires, string refreshToken)
+        public void SetToken(string accessToken, DateTime expires, string refreshToken)
```

Comment:
Technically this is a breaking change and should be versioned that way.  Is anything else using this method that will break when this package is updated or is this a method that was just recently added for this specific use case?  Would it be better to create an overload of this method that takes a dateTime?  Then we would not need to make a breaking change.

Reply:
@AQI-DanG  - this is a method that I added in this task. The change to the method was one of my commits